scripts/asm-installer/install_asm

```diff
@@ -692,6 +710,10 @@ EOF
     *) fatal "MODE must be one of 'install', 'migrate', 'upgrade'";;
   esac
 
+  if [[ "${MODE}" -ne "install" && "${ENABLE_HUB_IDNS}" -eq 1 ]]; then
```

Comment:
-ne only works for integer comparisons, to compare strings you need to use !=. (shell is weird)